psi4/src/psi4/optking/set_params.cc

```diff
@@ -314,7 +314,15 @@ void set_params(void)
       Opt_params.conv_max_disp  = 6.0e-5;  Opt_params.i_max_disp = true;
       Opt_params.conv_rms_disp  = 4.0e-5;  Opt_params.i_rms_disp = true;
     }
-    else if (Opt_params.general_conv == "GAU_VERYTIGHT") {
+    else if (Opt_params.general_conv == "INTERFRAG_GAU_TIGHT") {
+      Opt_params.i_untampered = true;
+      Opt_params.conv_max_DE    = 1.0e-6;  Opt_params.i_max_DE = true;
```

Comment:
since you're changing two convcrit (disp) and actually adding one (dE), I might advocate just `INTERFRAG_TIGHT` or  `INTERMOL_TIGHT`.